src/Pages/adminEdit/AdminEdit.js

```diff
@@ -0,0 +1,240 @@
+
+  const saveInput = e => {
+    const { name, value } = e.target;
+    setInput({ ...input, [name]: value });
+  };
+
+  const saveImage = e => {
+    setImageInput(e.target.files[0]);
+  };
+
+  const [dayNum, setDayNum] = useState(0);
+
+  const saveDayInput = e => {
+    const dayValue = Number(e.target.value);
+    if (e.target.checked) {
+      setDayNum(prev => prev + dayValue);
+    } else {
+      setDayNum(prev => prev - dayValue);
+    }
+  };
+
+  const [menus, setMenus] = useState([]);
+
+  const saveMenuInput = e => {
+    const { name, value } = e.target;
+    setMenuInput({ ...menuInput, [name]: value });
+  };
+
+  const [menuId, setMenuId] = useState(1);
+
+  const [menuInput, setMenuInput] = useState({
+    id: menuId,
+    name: '',
+    price: '',
+  });
+
+  const menuAddBtnClick = () => {
+    setMenuId(prev => prev + 1);
+    setMenus([...menus, menuInput]);
+    setMenuInput({ id: menuId + 1, name: '', price: '' });
+  };
+
+  const menuDeleteBtnClick = e => {
+    let newArr = [...menus];
+    const resultArr = newArr.filter(list => list.id !== Number(e.target.title));
+    setMenus(resultArr);
+  };
+
+  const adminEditForm = new FormData();
+  adminEditForm.append('name', input.name);
+  adminEditForm.append('address', input.address);
+  adminEditForm.append('tel', input.tel);
+  adminEditForm.append('open_time', input.open_time);
+  adminEditForm.append('closed_time', input.closed_time);
+  adminEditForm.append('image', imageInput);
+  adminEditForm.append('closed_day_id', dayNum);
+  adminEditForm.append('description', input.description);
+
+  const editSaveClick = () => {
+    fetch('https://b35e-211-106-114-186.jp.ngrok.io/store/admin', {
+      method: 'POST',
+      headers: {
+        enctype: 'multipart/form-data',
+        authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6OSwidXNlcl9pZCI6ImFhYWFhYWFhIiwiYWRtaW5URiI6InRydWUiLCJpYXQiOjE2NjM5MDc1MzB9.tGEtbRFN8RpDBfIqPwHQfzQm1aDJSqKpI9Lb51z_4cc',
+      },
+      body: adminEditForm,
+    })
+      .then(res => res.json())
+      .then(result => console.log(result));
+  };
+
+  return (
+    <div className="adminEdit">
+      <div className="adminEditBox">
+        <div className="adminEditTitle">내 가게 등록하기</div>
+        <div className="adminImageInputSet">
+          <div className="inputTitle">대표 사진</div>
+          <input type="file" accept="image/*" onChange={saveImage} />
+        </div>
+        {INPUT_VALUES.map(store => (
+          <div className="adminEditInputSet" key={store.id}>
+            <div className="inputTitle"> {store.title}</div>
+            <input
+              className="inputText"
+              type={store.type}
+              placeholder={store.placeholder}
+              name={store.name}
+              onChange={saveInput}
+              step={store.step}
+              min={store.min}
+              max={store.max}
+            />
+          </div>
+        ))}
```

Comment:
```suggestion
        {INPUT_VALUES.map(({id, title, type, placeholder, name, step, min, max}) => (
          <div className="adminEditInputSet" key={id}>
            <div className="inputTitle"> {title}</div>
            <input
              className="inputText"
              type={type}
              placeholder={placeholder}
              name={name}
              onChange={saveInput}
              step={step}
              min={min}
              max={max}
            />
          </div>
        ))}
```

추후 유지보수를 위해 위와 같은 방법으로 구조분해 할당 후 적용하는 것이 더 좋겠네요!